src/css/main.css

```diff
@@ -1 +1,132 @@
-/* General rules */
+*{
+    margin: 0;
+
+}
+header{
+    text-align: center;
+    height: 96vh;
+    border: 1px solid transparent;
+    background: url(https://web.archive.org/web/20140301004302im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg) 50% 100% no-repeat;
+}
+header h1{
+    color:#fff;
+    font-size: 3rem;
+    /* font-family: Helvetica; */
+    font-family: sans-serif;
+}
+header p{
+    color: #fff;
+    font-size: 35px;
+    margin-bottom: 35px;
+    font-family: sans-serif;
+}
+header .story{
+    font-family: sans-serif;
+    color:#fff;
+    font-size: 30px;
+    text-decoration: none;
+}
+header .story:hover{
+    text-decoration: underline;
+    
+}
+nav{
+    /* background: rgba(56, 56, 56, 0.534); */
+    /* background: rgba(95, 92, 92, 0.822); */
+    /* background: linear-gradient(rgba(0, 0, 0, 0.616),#fff); */
```

Comment:
#### `REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.